.github/workflows/update-dependencies.yml

```diff
@@ -45,4 +45,4 @@ jobs:
           branch: update-dependencies
           commit-message: "[Automated] Updating package dependencies"
           title: "[Automated] Update dependencies"
-          body: "Auto-generated update to the package dependencies. In order for this PR to be green, it will require all of the new dependencies to be mirrored to our AzDO NuGet feeds."
\ No newline at end of file
+          body: "Auto-generated update to the package dependencies. In order for this PR to be green, it will require all of the new dependencies to be mirrored to our AzDO NuGet feeds. Any updates to dependencies that begin with `System.*` or `Microsoft.*` should be reverted as those will be updated using regular dependency flow."
```

Comment:
what about `Microsoft.Azure.` `Microsoft.Data.SqlClient` `Microsoft.FluentUI` `Microsoft.Playwright` etc .. those won't be updated by codeflow?

Probably it should be a bit more explicit. Maybe it could be clearly commented in the directory.packages.props -- a separate commented section?

Reply:
They are already in their own separate section in Directory.Packages.props but I'll make this more explicit.